src/30_Advanced/SeatMap.html

```diff
@@ -79,6 +79,9 @@
           right: 0;
           margin: auto;
       }
+      .seatmap  amp-selector .svg-container {
+        padding: 0 20px 0 20px;
```

Comment:
shorter: `padding: 0 20px`.

Reply:
note, we usually use multiples of 8 for padding (0.5 rem). I that case I'd suggest using 24px (if it fits with the sample).

Reply:
done